src/_gettsim/functions_loader.py

```diff
@@ -104,15 +108,17 @@ def load_and_check_functions(
             user_functions,
             aggregation_functions,
             time_conversion_functions,
+            endogenous_groupings,
         ],
-        ["internal", "user", "aggregation", "time_conversion"],
+        ["internal", "user", "aggregation", "time_conversion", "endogenous_groupings"],
```

Comment:
```suggestion
        ["internal", "user", "aggregation", "time_conversion", "groupings"],
```

The `endogenous` is just for our understanding during the dev process.